liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/tagExists.test.groovy

```diff
@@ -34,10 +35,6 @@ Optional Args:
                 password:   { it.password },
                 tag: "version_2.0",
         ]
-
-        expectedResults = [
-                statusCode   : 0
-        ]
```

Comment:
Is the removal of this intentional?

Reply:
Yes, the status code is not returned when we change the implementation to AbstractCommandStep . I discussed this with Nathan and the idea is that if we have a failure an Exception will be thrown, so we don't need to expect anything here.